### PR TITLE
Syndicate Comms base now has GPS signal

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor
add: Spawning as a syndicate comms officer will now activate an encrypted signal in the Lavaland Syndicate Base, to aid the crew in retaliating.
/:cl:

I tied the GPS signal the voice changer so that it will only be active once the comms agent spawns (don't want it to be too easy for miners to locate an empty base and loot it)

I don't know how this makes any sense lorewise that their chameleon mask gives off a signal that miners can come track but hopefully it lets people raid the base more often when the comms agents start trolling the radio.

### CODE REVIEW

**Comment by RandomMarine** [comment]
I don't like this for the reasons brought up in #35019

**Comment by Dorsisdwarf** [comment]
Make the station have to buy a syndicate decryptor from a hacked (not emagged) cargo console for a load of points to locate it so theres some effort involved

**Comment by KorPhaeron** [comment]
The effort is digging across lavaland to cross the giant lava moat and to dodge the turrets and sniper rifles

**Comment by Kromgar** [comment]
@Dorsisdwarf 

>Effort

>Ignores invading a base full of well armed agents equipped with sniper rifles

>Kill turrets

>Bombs


>"Needs some effort"

**Comment by 81Denton** [comment]
That GPS signal might as well be named "valids and loot here".

@Kromgar 
Grab a firefighter APLU and just drill through the wall. The turrets are a joke and syndies are boxed in after all.

**Comment by Durkel** [comment]
If they're meant to be a listening post wouldn't it make more sense to just remove their ability to communicate outside of the traitor channel?  You're not a very good listening post if you're too busy being a jack ass on the radio 24/7.

**Comment by WJohn** [comment]
I'd be cool with limiting him to *just* contacting people on the syndicate radio.

**Comment by improvedname** [comment]
Just tie it to an gps ghost roles need to stop being wimps

**Comment by Jalleo** [comment]
I've removed the voice mask in my PR and role from space spawn for the time being. Really long term we should have a multi area (Syndies that can go to a couple of ruins) for different tasks and divsersify the tasks they can do. Thirdly manipulating records in medical and so forth is probably fine as that would be more fluff as nobody really uses it.

**Comment by Slignerd** [comment]
Tweak it to instead activate when bluespace transmitter is activated.

File: code/modules/ruins/lavaland_ruin_code.dm [hunk 1]
```diff
@@ -127,6 +127,16 @@
...
+
+/obj/item/clothing/mask/chameleon/gps/Initialize()
+	. = ..()
+	new /obj/item/device/gps/internal/lavaland_syndicate_base(src)
+
+/obj/item/device/gps/internal/lavaland_syndicate_base
+	icon_state = null
+	gpstag = "Encrypted Signal"
+	desc = "A signal from an unidentifed broadcast."
+	invisibility = 100
```
Comment (Cyberboss) [inline code/modules/ruins/lavaland_ruin_code.dm]: The only var you shoud set is gpstag

**Review by Cyberboss — DISMISSED** [review]

**Comment by Cyberboss** [comment]
Changelog

**Comment by Robustin** [comment]
I don't play Lavaland much but someone needs to explain how a base with turrets, self-destruct devices that can be set off instantly, and multiple heavily armed syndicate defenders is even remotely a "free loot" experience. 

Plus unless the miners get lucky with the chest they don't have anything that can contest the sniper/stechkin wielding syndies. Plus once you get inside all someone has to do is snip a wire and the entire region becomes a crater. As far as risk/reward goes for Lavaland, that seems fair and provides the possibility for entertaining conflict besides someone just spamming comms indefinitely.

**Comment by 81Denton** [comment]
I didn't notice that, my bad.

**Comment by F-OS** [comment]
If it could only activate when you use the PDA messenger/start talking shit outside of syndicate comms that would be amazing.

**Comment by Slignerd** [comment]
@KorPhaeron The problem is that your presence is confirmed as soon as someone grabs a GPS.

**Comment by Jalleo** [comment]
@Slignerd only if you spawn un as a comms guy. Until then no gps.

**Comment by KorPhaeron** [comment]
@Cyberboss

**Comment by Slignerd** [comment]
@Jalleo Yes, that's what I mean/

**Comment by duncathan** [comment]
now that there is only a 10% chance of appearance I don't believe this is necessary

**Comment by KorPhaeron** [comment]
This is for another ruin dunc

**Comment by duncathan** [comment]
my b

**Pull request merged**